fabric-biomes-v1/build.gradle

```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-biomes-v1"
-version = getSubprojectVersion(project, "0.1.5")
+version = getSubprojectVersion(project, "0.1.5")
```

Comment:
Where did the newline go?

Reply:
Thonk

Reply:
We should probably increment the version here and restore the newline

Reply:
increment to what?

Reply:
0.2.0 (added features)